### PR TITLE
add convention for feedback github repo

### PR DESCRIPTION
**Pull request by OsmondJiang**
@mingwli , maybe we need change migration tool

### CODE REVIEW

**Comment by OsmondJiang** [comment]
image
Want to avoid that

File: docs/specs/localization.yml [hunk 1]
```diff
@@ -1093,3 +1093,19 @@ repos:
...
     ["error","config-not-found","Cannot find 'docfx.yml/docfx.json' at *"]
+---
+# feedback github repo
+commands:
+  - build --locale zh-cn
+repos:
+  https://github.com/feedback/a:
+    - files:
+        docfx.yml: |
+          feedbackGitHubRepo: microsoftdocs/docfx-docs-pr
```
Comment (mingwli) [inline docs/specs/localization.yml]: >feedbackGitHubRepo: microsoftdocs/docfx-docs-pr [](start = 10, length = 47)

need we consider branch mapping?

Comment (OsmondJiang) [reply docs/specs/localization.yml]: githubRepo doesn't have branch info

File: src/docfx/build/legacy/metadata/LegacyMetadata.cs [hunk 2]
```diff
@@ -144,6 +144,11 @@ public static JObject GenerateLegacyRedirectionRawMetadata(Docset docset, PageMo
                     rawMetadata["original_content_git_url"] = pageModel.OriginalContentGitUrl;
             }
 
+            if (string.IsNullOrEmpty(pageModel.FeedbackGitHubRepo))
```
Comment (OsmondJiang) [inline src/docfx/build/legacy/metadata/LegacyMetadata.cs]: !

File: src/docfx/config/Config.cs [hunk 3]
```diff
@@ -146,7 +146,13 @@ internal sealed class Config
         public readonly string Theme = string.Empty;
 
         /// <summary>
-        /// Gets the config file name.
+        /// Get the feedback github repo({org}/{name}) like "dotnet/docfx"
+        /// It's used for legacy doc(docs.com) sites build
+        /// </summary>
+        public readonly string FeedbackGitHubRepo = string.Empty;
```
Comment (superyyrrzz) [inline src/docfx/config/Config.cs]: group config by feature: move to `gitHub.feedbackRepo`?

**Comment by yufeih** [comment]
`feedback_github_repo` is not a docfx config, this approach does not scale to other metadatas

**Pull request closed without merging**